app/models/role.rb

```diff
@@ -46,6 +46,18 @@ def current_holder_biography
     current_holder["details"]["body"]
   end
 
+  def past_holders
```

Comment:
Since we now have methods for fetching the previous and current role appointments in this class (`current_holder` and `past_holders`) it might be better to generalise it a bit like I did for people: https://github.com/alphagov/collections/blob/4be621d6d5d4a8358e21cea78269590a8636b691/app/models/person.rb#L93-L106

I also wonder (maybe not for this PR) but whether it might make sense to have some sort of mixin class like `HasRoleAppointments` which we can `include` in both this class and the person class. But on the other hand this might be confusing as the logic is then somewhere else. 🤷‍♂